SourceX/miniwin/misc_msg.cpp

```diff
@@ -187,7 +187,7 @@ static WPARAM keystate_for_mouse(WPARAM ret)
 
 static WINBOOL false_avail()
 {
-	DUMMY_PRINT("return false although event available", 1);
+	DUMMY_PRINT("return false although event available");
```

Comment:
This causes a build error for MSVC

Reply:
Ok, gcc got an extension for empty variadic macro (https://gcc.gnu.org/onlinedocs/gcc/Variadic-Macros.html - "empty, the ‘##’ operator causes the preprocessor to remove the comma before it." ) that MSVC do not have. So I'll remove the commit